eyeshade/workers/publishers.js

```diff
@@ -46,9 +44,6 @@ exports.workers = {
           throw e
         }
         await client.query('COMMIT')
```

Comment:
settlement reports run very infrequently (~2 times a month). I think we should keep the refresh here

Reply:
i mean, yes, but it happens continuously